interface/themes/select2-bootstrap-theme.scss

```diff
@@ -0,0 +1,172 @@
+/*
+* MIT License
+*
+* Copyright (c) 2018 Takashi Kanemoto
+
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+/*
+* Select2 BS4 Theme without BS4 imports mod (BS4 imports in current build cause errors)
+* Tested on version 1.3.2
+*/
+
+/* TODO: Couldn't get this imported with public assets for some reason */
+@import "../../node_modules/select2-bootstrap4-theme/src/single";
+@import "../../node_modules/select2-bootstrap4-theme/src/multiple";
```

Comment:
this above stuff needs to to be brought into public/assets

Also, where did this script originally come from?

Reply:
Original script from https://github.com/ttskch/select2-bootstrap4-theme/blob/master/src/layout.scss
Whoever coded it messed me up cause they put the bootstrap imports in which I didn't need. For some reason while compiling and attempting to move it to public/assets it didn't work. Probably cause too many arguments or something.

Reply:
Let me know if you want me to have gulp bring in select2-bootstrap4-theme/src to public/assets (now it just brings in select2-bootstrap4-theme/dist). Would it also make sense for gulp to bring over src/layout.scss to interface/themes with the mods (ie. replace the imports ;then easier to keep things updated as new select2-bootstrap4-theme versions come out)?

Reply:
Better yet, could have gulp directly modify the select2-bootstrap-theme.scss in public/assets/../src with the adjusted imports.

Reply:
You know what that might be a good idea so I can just import the code in one line in the core scss rather than creating an entirely separate file. I'm not quite sure how to fix it to delete the 3 lines I don't need though and it has to be done before compiling to prevent errors. Also moving it to public/assets would be great

Reply:
Sounds good. I'll work on that (should get it into this PR within next 2 days).

Reply:
As an aside, the select2 use in Administration->Forms->Lists is strange looking. Unclear the item is even a list and when click on it am unable to scroll down the list (it just cuts off at bottom of screen).

Reply:
Just swapped the theme for select2 to BS4 in that file and it isn't behaving nicely. Not too sure what's happening there.

Reply:
@bradymiller So I found something interesting... when I load up the dark theme the select2 BS4 theme loads up correctly but when I load up the light theme it's as if it isn't even there for some reason....